dts/riscv/it8xxx2.dtsi

```diff
@@ -559,6 +572,14 @@
 			status = "disabled";
 			label = "ADC_0";
 			#io-channel-cells = <1>;
+			pinctrls = <&pinmuxi 0 IT8XXX2_PINMUX_FUNC_1   /* ADC0*/
+				    &pinmuxi 1 IT8XXX2_PINMUX_FUNC_1   /* ADC1*/
```

Comment:
Could you structure this to have a single handle with the specific pin combination? For example for npcx has all possible combinations in https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/nuvoton/npcx/npcx7-alts-map.dtsi, similarly stm32 in their own hal repository (https://github.com/zephyrproject-rtos/hal_stm32/blob/master/dts/st/f0/stm32f030cctx-pinctrl.dtsi). The file (I assume is generated) should contain all the possible combinations with a meaningful mnemonic, so that once it's done you would not need the reference manual to find what function you need, and it makes the comment redundant as well (imagine this being like &adc1_pi1, instead of the tuple and comment).

Reply:
I agree with this approach.  Please also name the property "pinctrl-0" to match other driver definitions.

Reply:
Done